### ISSUE TITLE
Docker WSL2 changing UUID after restarts - UUID not consistent

### ISSUE THREAD

**Issue by bakerboy448** [Notifiarr/notifiarr#129]
Running the client on WSL2 results in a new client on restart - @austinwbest put a temp workaround in to ignore it.

Reported via Discord via @sadstan

> without going deep enough to learn GO and the kernel, the UUID is the same as boot_id, which changes between reboots

- Rox

**Comment by davidnewhall** [comment]
> The Windows Subsystem for Linux lets developers run a GNU/Linux environment -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a traditional virtual machine or dualboot setup.

This is not a supported configuration. If a developer wants to run this app on WSL, they should very well know how to develop their way out of the bugs associated with such a system.

**Comment by davidnewhall** [comment]
Related:
https://github.com/giampaolo/psutil/issues/1251
https://github.com/shirou/gopsutil/pull/994

tl;dr: the host UUID changes after every reboot. This behavior appears to only exist in WSL.

**Comment by davidnewhall** [comment]
This only affects Docker on WSL. Solution: Map `/etc/machine-id` with a volume.
```
-v /etc/machine-id:/etc/machine-id
```

**Comment by austinwbest** [comment]
I removed the host ID from the uuid builder for wsl2 platforms.

**Comment by austinwbest** [comment]
For future users (if any) 

https://notifiarr.wiki/en/Client/Installation#wsl2-users
